Userland/Libraries/LibCpp/Preprocessor.h

```diff
@@ -38,13 +38,13 @@ class Preprocessor {
 
 private:
     using PreprocessorKeyword = StringView;
-    PreprocessorKeyword handle_preprocessor_line(const StringView&);
-    void handle_preprocessor_keyword(const StringView& keyword, GenericLexer& line_lexer);
+    PreprocessorKeyword handle_preprocessor_line(StringView const&);
+    void handle_preprocessor_keyword(StringView const& keyword, GenericLexer& line_lexer);
+    Vector<Token> process_line(StringView const& line);
 
-    Definitions m_definitions;
     const String m_filename;
-    const StringView m_program;
-    StringBuilder m_builder;
+    const String m_program;
```

Comment:
`const String` seems odd, did you mean `const String&`?
(also above for the filename)

Reply:
if you _did_, please consider switching to `StringView`

Reply:
Yeah the `const` on those Strings doesn't make much sense, I remove them.